xml/schema/Core/Phone.xml

```diff
@@ -148,7 +148,7 @@
   </field>
   <field>
     <name>phone_type_id</name>
-    <title>Phone Type</title>
+    <title>Phone Type ID</title>
```

Comment:
So I'm a -1 on this change and would prefer to override the label in `exportableFields` or wherever it needs to contain ID but leave it user-friendly in the metadata.

Whichever way we go we should be consistent with all fields, and at this point most titles in the metadata do *not* have the "ID" appended.

Reply:
@colemanw so the fields are ID but the pseudoconstants all have names which make sense for the entity reference etc

Reply:
So for example the fields array winds up with 2 fields - Phone Type - which has a label from the option_group.label & phone_type_id which is the integer. This also manifests in qf-search builder & I assume views filters on an integer not a  string when you add it in the url. We have got a few fields that are pretty confusing in search builder now as the labels got renamed so the titles are duplicated - but I think we should use the pseudoconstant titles & let the fields have the correct name